monorepo-folders/README.md

```diff
@@ -0,0 +1,70 @@
+# monorepo-folders
+
+This example is meant to demonstrate a realistic monorepo folder structure where:
+
+- `frontend-ui` contains the frontend UI that calls some backend APIs (we use Create-React-App)
+- `backend-apis` contains all the backend API routes (we use Express.js here, but could easily be a set of serverless functions)
+  - Some routes have a Temporal Client that calls workflows
+- `temporal-worker` has independently scalable Workers that have Workflows and Activities registered
+- `temporal-workflows` is a shared source folder containing Workflow and Activity code that is used by both the Client in `backend-apis` and the Worker in `temporal-worker`
+
+```bash
+packages/
+  backend-apis/
+    server.ts # runs Express server on localhost:4000 
+  frontend-ui # this is a Create-React-App on localhost:3000, proxies api requests to :4000
+  temporal-worker/
+    worker.ts # registers Temporal Worker that has Workflows and Activities from /temporal-workflows
+  temporal-workflows/
+    lib/ # compiled workflows
+    src/
+      workflowA/
+          activities/
+            activitiesA.ts
+            activitiesB.ts
+          workflow.ts
+      workflowB/
+          activities/
+            activitiesC.ts
+            activitiesD.ts
+          workflow.ts
+      all_workflows.ts
+      all_activities.ts
+node_modules
+package.json
+```
+
+Notes on the structure demonstrated:
+
+- **Workflows require one file**:  you can organize Workflow code however you like, but each Worker needs to reference a single file that exports all the Workflows it handles (so that the Worker's webpack has an entry point)
```

Comment:
1 file is required because workflows are registered when exported, if we let devs use multiple files we’d have to handle name conflicts instead of letting js do that for us

Reply:
```suggestion
- **Workflows require one file**:  you can organize Workflow code however you like, but each Worker needs to reference a single file that exports all the Workflows it handles (so you have to handle name conflicts instead of us)
```